docs/user_guide.rst

```diff
@@ -10,18 +10,39 @@ PyNWB to solve the challenges of variety of data formats and size of data,
 NeuroConv build upon these automated conversion tools to provide a
 system for combining data across multiple streams.
 
-This workflow is as follows:
+Any conversion task requires three sets of information:
 
-#. Extract metadata from existing proprietary files
-#. Adjust or add any missing metadata
-#. Run the conversion, inserting this metadata in the appropriate place.
+#. **Source data**: The paths to existing proprietary files and directories
+#. **Metadata**: The metadata that can be fetched from source data or added manually
+#. **Conversion options**: The configuration to run conversion
```

Comment:
@CodyCBakerPhD 
Overall, I decided to extend the User Guide with the content found in `documentation/neuroconv_structure.md`. Let me know if you think they should go in a different page. Also I'm not sure whether to include the ["Step by step operations"](https://github.com/catalystneuro/neuroconv/blob/main/documentation/neuroconv_structure.md#step-by-step-operations), I decided that for real examples I would point the reader to the conversion gallery and make this user guide a more detailed description of interfaces and converters.

Reply:
let me know if you like it, then I can remove the markdown file (the figure is already moved to docs/img) and edit the changelog.

Reply:
Or I suppose the `nwb_conversion_tools` reference I spotted is maybe not in this changelog but noticed it here: https://neuroconv--39.org.readthedocs.build/en/39/user_guide.html#basedatainterface

Reply:
ah you're right, I didn't notice that one.